### PR TITLE
feat: Show multiline encoding option during secret creation

### PR DESCRIPTION
**Pull request by ShubhamPalriwala**
# Description 📣
Add Option to enable multi line encoding during secret creation input form

Demo Video: https://www.tella.tv/video/shubham-palriwalas-video-8t4s

## Type ✨

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/getting-started/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/getting-started/code-of-conduct). 📝

### CODE REVIEW

File: frontend/src/views/SecretOverviewPage/components/CreateSecretForm/CreateSecretForm.tsx [hunk 1]
```diff
@@ -59,6 +62,13 @@ export const CreateSecretForm = ({
     formState: { isSubmitting, errors }
   } = useForm<TFormSchema>({ resolver: zodResolver(typeSchema) });
   const newSecretKey = watch("key");
+  const newValue = watch("value");
```
Comment (sheensantoscapadngan) [inline frontend/src/views/SecretOverviewPage/components/CreateSecretForm/CreateSecretForm.tsx:65]: nit: let's be more specific with the name? maybe "newSecretValue"

**Review by sheensantoscapadngan — COMMENTED** [review]
should we also add this to the main page?
image

File: frontend/src/views/SecretOverviewPage/components/CreateSecretForm/CreateSecretForm.tsx (hunk 1 above)
Comment (ShubhamPalriwala) [reply frontend/src/views/SecretOverviewPage/components/CreateSecretForm/CreateSecretForm.tsx]: thanks, made the change

**Comment by ShubhamPalriwala** [comment]
Thanks @sheensantoscapadngan, I've made the changes you recommended. Can you please re-review it once?

**Comment by sheensantoscapadngan** [comment]
I think multi-line encoding should be disabled by default? 
image

**Pull request closed without merging**